deploy/charts/harvester/questions.yaml

```diff
@@ -347,140 +347,6 @@ questions:
-        type: "boolean"
-        group: "Minio Settings"
-      - variable: minio.persistence.storageClass
-        default: ""
-        label: "Storage Class"
-        description: "specify the storage class of persistence volume claim, use default if blank"
-        type: "storageclass"
-        group: "Minio Settings"
-        show_if: "minio.persistence.enabled=true && longhorn.enabled=false"
-      - variable: minio.persistence.size
-        default: "20Gi"
-        label: "Storage Size"
-        description: "specify the size of persistence volume claim"
-        type: "string"
-        group: "Minio Settings"
-        show_if: "minio.persistence.enabled=true"
-  #################### Harvester VM Image Storage Gateway Settings ####################
-  - variable: containers.apiserver.vmImageStorageGateway.endpoint
-    default: ""
-    label: "Endpoint of VM Image Storage"
-    description: "specify the endpoint of vm image storage gateway"
-    type: "string"
-    group: "Harvester VM Image Storage Gateway Settings"
-    show_if: "minio.enabled=false"
-    required: true
-  - variable: select_harvester_vm_image_storage_gateway_mode
-    default: "From-Secret"
-    label: "Get Account of VM Image Storage"
-    description: "specify how to get the account of vm image storage gateway"
-    type: "enum"
-    options:
-      - "From-Secret"
-      - "From-Input"
-    group: "Harvester VM Image Storage Gateway Settings"
-    show_if: "minio.enabled=false"
-  - variable: containers.apiserver.vmImageStorageGateway.secretName
-    default: ""
-    label: "Secret Name of Account"
-    description: "specify the name of the same namespace existing Secret instance, which stored the access key(in 'accesskey' field) and secret key(in 'secretkey' field)"
-    type: "secret"
-    group: "Harvester VM Image Storage Gateway Settings"
-    show_if: "minio.enabled=false && select_harvester_vm_image_storage_gateway_mode=From-Secret"
-    required: true
-  - variable: containers.apiserver.vmImageStorageGateway.accessKey
-    default: ""
-    label: "Access Key of Account"
-    description: "specify the access key, the value will be stored in 'accesskey' field of a specified secret"
-    type: "string"
-    group: "Harvester VM Image Storage Gateway Settings"
-    show_if: "minio.enabled=false && select_harvester_vm_image_storage_gateway_mode=From-Input"
-    required: true
-  - variable: containers.apiserver.vmImageStorageGateway.secretKey
-    default: ""
-    label: "Secret Key of Account"
-    description: "specify the secret key, the value will be stored in 'secretkey' field of a specified secret"
-    type: "string"
-    group: "Harvester VM Image Storage Gateway Settings"
-    show_if: "minio.enabled=false && select_harvester_vm_image_storage_gateway_mode=From-Input"
-    required: true
   #################### Harvester Storage Settings ####################
   - variable: longhorn.enabled
     default: "true"
```

Comment:
can we set `longhorn.enabled` default to true in the `values.yaml` and update the chart install's README file since longhorn becomes a required dependency of the Harvester for now.

Reply:
Updated